code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -54,14 +54,23 @@
 	list_reagents = list("omnizine" = 30)
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF
 
+/obj/item/reagent_containers/hypospray/emergency_combat
+	name = "emergency combat stabilizer" //Used for the new combat medical kit.
+	desc = "A small modified air-needle auto injector, used by support operatives to quickly stabilize their allies in critical condition."
+	amount_per_transfer_from_this = 5
+	icon_state = "hypo"
+	volume = 35
+	ignore_flags = 1 // So they can heal their comrades.
+	list_reagents = list("oculine" = 5, "stimulants" = 15, "atropine" = 15)

```

Comment:
oculine is useless to nukies, they are flash proof so they cant take eye damg

Reply:
I've mistaken Inacusiate with oculine, that's truly an error on my side.